src/dodal/devices/focusing_mirror.py

```diff
@@ -51,9 +50,9 @@ class SingleMirrorVoltage(Device):
     """
 
     def __init__(self, name: str = "", prefix: str = ""):
-        self._actual_v = epics_signal_r(int, prefix + "R")
-        self._setpoint_v = epics_signal_rw(int, prefix + "D")
-        self._demand_accepted = epics_signal_r(MirrorVoltageDemand, prefix + "DSEV")
+        self.actual_v = epics_signal_r(int, prefix + "R")
+        self.setpoint_v = epics_signal_rw(int, prefix + "D")
+        self.demand_accepted = epics_signal_r(MirrorVoltageDemand, prefix + "DSEV")
```

Comment:
Should: As above, I think keep these private to avoid people setting then directly